sources/packages/backend/libs/integrations/src/services/disbursement-schedule/disbursement-schedule.service.ts

```diff
@@ -199,4 +200,68 @@ export class DisbursementScheduleService extends RecordDataModelService<Disburse
     const repository = disbursementScheduleRepo ?? this.repo;
     return repository.update({ id: In(disbursementIds) }, { dateSent });
   }
+
+  /**
+   * Fetch the eligible COEs for a given date.
+   * @param generatedDate Date in which the eligible COE is required for an institution.
+   * @returns eligible COEs.
+   */
+  async getEligibleCOE(
+    generatedDate?: string,
+  ): Promise<DisbursementSchedule[]> {
+    const processingDate = generatedDate
+      ? new Date(generatedDate)
+      : addDays(-1);
+    return this.repo.find({
```

Comment:
The query must consider only assessments associated with the `application.currrentAssessment`.
Please review the query used to display the COEs for the institutions, both must be in sync.